src/panels/lovelace/hui-root.ts

```diff
@@ -377,30 +375,36 @@ class HUIRoot extends LitElement {
                             )}
                             <ha-svg-icon
                               slot="graphic"
-                              .path=${mdiCog}
+                              .path=${mdiPencil}
                             ></ha-svg-icon>
                           </mwc-list-item>
                         `
                       : ""}
-                    <a
-                      href=${documentationUrl(this.hass, "/lovelace/")}
-                      rel="noreferrer"
-                      class="menu-link"
-                      target="_blank"
-                    >
-                      <mwc-list-item
-                        graphic="icon"
-                        aria-label=${this.hass!.localize(
-                          "ui.panel.lovelace.menu.help"
-                        )}
-                      >
-                        ${this.hass!.localize("ui.panel.lovelace.menu.help")}
-                        <ha-svg-icon
-                          slot="graphic"
-                          .path=${mdiHelp}
-                        ></ha-svg-icon>
-                      </mwc-list-item>
-                    </a>
+                    ${this._editMode
```

Comment:
This doesn't match your screenshot. This is only showing the help link _inside_ the overflow menu when in edit mode.

Shouldn't we just kill the overflow menu in edit mode?

Reply:
Assuming you mean this: 
![image](https://user-images.githubusercontent.com/114137/149615318-b4668130-58f3-4c98-bee8-f43857c12489.png)

Why kill it? I think the four entries in the edit mode overflow menu are worthwhile and should be kept.

Reply:
oh my bad. forgot about those.